code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -373,3 +375,4 @@
 			dat += "[new_text]\n" //dat.Join("\n") doesn't work here, for some reason
 	if(dat.len)
 		return dat.Join()
+a
```

Comment:
I'm in awe at BYOND at the fact that this is compiling

Reply:
```suggestion

```

Reply:
Lmao what the fuck

Reply:
It compiles because it's defining a new atom of type `a`.